iothub/device/src/Common/CommonConstants.cs

```diff
@@ -85,6 +85,10 @@ internal static class CommonConstants
         // Security message constants
         public const string SecurityMessageInterfaceId = "urn:azureiot:Security:SecurityAgent:1";
 
+        // Component constants
```

Comment:
nit: this is more impl thing but we should call out that these ae identifiers for property constants used for pnp based device property patches